commit 0fadf0a3baee106b0eb12c32f9e49140c5355221 (refs/changes/97/397/1)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2016-09-02T01:18:20-07:00 (3 years, 1 month ago)
    
    DOCKER-923 sdc-docker api-create tests checks for incorrect property on error object when determining whether fabrics are enabled

diff --git a/test/integration/api-create.test.js b/test/integration/api-create.test.js
index da3c40f..a98b14c 100644
--- a/test/integration/api-create.test.js
+++ b/test/integration/api-create.test.js
@@ -353,7 +353,7 @@ test('ensure fabrics enabled', function (tt) {
             function (err, vlans) {
                 if (err) {
                     FABRICS = false;
-                    if (err.code !== 'PreconditionRequiredError') {
+                    if (err.restCode !== 'PreconditionRequiredError') {
                         t.ifErr(err);
                     }
                 } else {
